tokens/src/lib.rs

```diff
@@ -358,6 +358,24 @@ pub mod module {
 			who: T::AccountId,
 			amount: T::Balance,
 		},
+		Deposit {
```

Comment:
there is already Deposited, Withdrawn etc

Reply:
https://github.com/open-web3-stack/open-runtime-module-library/pull/931/commits/4c851e507c7e3b2b6527aa30a2e9402ab6f22655